omnibus/files/private-chef-cookbooks/private-chef/recipes/rabbitmq.rb

```diff
@@ -186,12 +186,20 @@
   execute "#{rmq_plugins} enable rabbitmq_management" do
     environment (rabbitmq_env)
     user opc_username
-    not_if "#{rmq_plugins} list | grep rabbitmq_management | grep -v rabbit_management_agent | grep E"
+    not_if { !rabbitmq['management_enabled'] || shell_out("#{rmq_plugins} list | grep rabbitmq_management | grep -v rabbit_management_agent | grep E").status.success? }
```

Comment:
I think it is possible to put two not_if attributes on the resource, might be a bit cleaner:

```
not_if { !rabbitmq['management_enabled'] }
not_if "#{rmq_plugins} list | grep rabbitmq_management | grep -v rabbit_management_agent | grep E"
```

A further cleanup would be to to put the command that checks whether rabbitmq is enabled in a library so it could be:

```
not_if  { !rabbitmq['management_enabled'] || RabbitHelper.plugin_enabled?("rabbitmq_management") }
```

but that is probably a bit much for this small fix.